src/context.js

```diff
@@ -96,6 +147,9 @@ function EvolvContext(store) {
     }
 
     objects.setKeyToValue(key, value, context);
+    if (this.isPersisted(key)) {
```

Comment:
Does anything ever get persisted? From the looks of the code, isPersisted will only ever return true if we call persistValue -- which is inside this if